pyproject.toml

```diff
@@ -54,7 +54,7 @@ pandas = "^2.0.3"
 platformdirs = "^3.5.1"
 psutil = "^5.9.5"
 pyyaml = "^6.0"
-requests = "^2.31.0"
+requests = "==2.31.0"
```

Comment:
It has to be this exact version, right?

Reply:
`"^2.31.0"` was looking for 2.32.0, `"==2.31.0"` will look  for exact version.